.travis.yml

```diff
@@ -11,6 +11,6 @@ sudo: false
 env:
 - TERM=dumb # Makes Gradle use plain console output
 script:
-- mvn clean install
-- mvn checkstyle:check
+- mvn clean install -B -Dstyle.color=always
```

Comment:
Looks like the -B reduces the number of lines in Travis output from 15k to 3k. If I'm understanding right. It seems to mitigate the problem where Travis has to spin for a while before it can show the build log.